src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -26,6 +26,7 @@
 
   protected int sentinelConnectionTimeout;
   protected int sentinelSoTimeout;
+  protected String sentinelUser;
```

Comment:
Hmm I missed this when implementing. (the fact that we can have different user for db and sentinel)

Should we add this in the test suite? (have a specific user for sentinel)

Reply:
Comment in JedisClusterInfoCache, Line 205 applies here as well.